### PR TITLE
add $envelopeNamespace argument to Security construct (SOAP 1.2 support)

### PR DESCRIPTION
**Pull request by NtlBldrv**
Hi! Added SOAP 1.2 support, please, consider adding in to a new release, it might be helpful for others

### CODE REVIEW

**Comment by NtlBldrv** [comment]
@mikaelcom done!

File: src/Security.php [hunk 1]
```diff
@@ -13,12 +13,12 @@ class Security extends Element
...
      * @var string
      */
-    const ATTRIBUTE_MUST_UNDERSTAND = 'SOAP-ENV:mustunderstand';
+    const ATTRIBUTE_MUST_UNDERSTAND = ':mustunderstand';
     /**
      * Element attribute mustunderstand name
      * @var string
      */
-    const ATTRIBUTE_ACTOR = 'SOAP-ENV:actor';
+    const ATTRIBUTE_ACTOR = ':actor';
```
Comment (mikaelcom) [inline src/Security.php]: Let's introduce the constant for outside usage
```suggestion
    const ATTRIBUTE_ACTOR = ':actor';
    /**
     * Envelop namespace
     * @var string
     */
    const ENV_NAMESPACE = 'SOAP-ENV';
```

File: src/WsSecurity.php [hunk 2]
```diff
@@ -8,35 +8,50 @@ class WsSecurity
...
+        $username,
+        $password,
+        $passwordDigest = false,
+        $addCreated = 0,
+        $addExpires = 0,
+        $mustunderstand = false,
+        $actor = null,
+        $usernameId = null,
+        $addNonce = true,
+        $envelopeNamespace = 'SOAP-ENV'
```
Comment (mikaelcom) [inline src/WsSecurity.php]:
```suggestion
        $envelopeNamespace = Security::ENV_NAMESPACE
```

File: src/WsSecurity.php [hunk 3]
```diff
@@ -8,35 +8,50 @@ class WsSecurity
...
     /**
-     * @param bool $mustunderstand
+     * @param bool   $mustunderstand
      * @param string $actor
+     * @param string $envelopeNamespace
+     *
      * @return WsSecurity
      */
-    protected function initSecurity($mustunderstand = false, $actor = null)
+    protected function initSecurity($mustunderstand = false, $actor = null, $envelopeNamespace)
```
Comment (mikaelcom) [inline src/WsSecurity.php]: In my opinion, having a method with the two first parameters as optional should not have a third required parameter that would require to set the two first parameters anytime.
```suggestion
    protected function initSecurity($mustunderstand = false, $actor = null, $envelopeNamespace = Security::ENV_NAMESPACE)
```

File: src/WsSecurity.php [hunk 4]
```diff
@@ -8,35 +8,50 @@ class WsSecurity
...
     {
-        $this->security = new Security($mustunderstand, $actor);
+        $this->security = new Security($mustunderstand, $actor, $envelopeNamespace);
```
Comment (mikaelcom) [inline src/WsSecurity.php]: According to my previous suggestion
```suggestion
        $this->security = new Security($mustunderstand, $actor, Security::NS_WSSE, $envelopeNamespace);
```

File: src/WsSecurity.php [hunk 5]
```diff
@@ -46,22 +61,38 @@ public function getSecurity()
...
+        $password,
+        $passwordDigest = false,
+        $addCreated = 0,
+        $addExpires = 0,
+        $returnSoapHeader = true,
+        $mustunderstand = false,
+        $actor = null,
+        $usernameId = null,
+        $addNonce = true,
+        $envelopeNamespace = 'SOAP-ENV'
```
Comment (mikaelcom) [inline src/WsSecurity.php]: Did you know I can be annoying :smile:?

```suggestion
        $envelopeNamespace = Security::ENV_NAMESPACE
```

**Comment by NtlBldrv** [comment]
Totally missed this one! Thanks for pointing out!

**Pull request merged**